docs/user_guide/hopsworks/search.rst

```diff
@@ -49,6 +49,7 @@ Results will also have highlighted the reason why they matched your search query
 	* Name
 	* Description
 	* Tags
+    * Features
```

Comment:
Identation?

Reply:
fixed... it was the space vs tab issue again..